src/pages/SignUp/index.tsx

```diff
@@ -30,14 +32,17 @@ function SignUp() {
   } = useForm({ resolver: yupResolver(signUpSchema) });
 
   const onSubmit: SubmitHandler<SignUpInputsType> = async (data: SignUpInputsType) => {
-    try {
-      await dispatch(signUp(data)).unwrap();
-      navigate(RoutherPaths.LOG_IN);
-    } catch (e) {
-      console.log(e);
-    }
+    await dispatch(signUp(data)).unwrap();
+    navigate(RoutherPaths.LOG_IN);
   };
 
+  useEffect(() => {
+    if (isAuthenticated) navigate(RoutherPaths.HOME);
```

Comment:
according to isAuthenticated comment you can move this guy out from this useEffect too :)